front/public/locales/en/translation.json

```diff
@@ -447,7 +447,8 @@
         "remove-track-range": "Click to detatch from:",
         "speed-limits": "Speed limits",
         "toggle-psl": "Permanent speed limit",
-        "toggle-psl-help": "Click on a track section first to be able to activate this option"
+        "toggle-psl-help": "Click on a track section first to be able to activate this option",
+        "add-new-tag": "Add a new composition code"
```

Comment:
Keys need to be sorted alphabetically (same for fr file)

Reply:
Done